CMakeLists.txt

```diff
@@ -86,6 +101,7 @@ endif()
 if(EXISTS ${XGL_SPVGEN_PATH})
     set(XGL_SPVGEN_BUILD_PATH ${CMAKE_BINARY_DIR}/spvgen)
     add_subdirectory(${XGL_SPVGEN_PATH} ${XGL_SPVGEN_BUILD_PATH} EXCLUDE_FROM_ALL)
+    target_link_libraries(spvgen_base vkgc-headers) # TODO Remove after added in spvgen
```

Comment:
I'm not sure what the naming conventions are exactly. IIRC llvm libraries tend to use `-` while most other projects use `_`. Since vkgc is considered outside of LGC (I think?), shouldn't it use underscores in library names?
I don't care much either way, but I have a slight preference for things being consistent.